web-ui/src/main/resources/catalog/components/elasticsearch/directives/FacetDirective.js

```diff
@@ -184,6 +184,18 @@
           };
         }
       };
+
+      this.sortByQuantity = function (agg, bucket) {
+        return function (facet) {
+          return facet.count;
+        };
+      };
+
+      this.sortByQuantityDesc = function (agg, bucket) {
+        return function (facet) {
+          return -1 * facet.count;
+        };
+      };
```

Comment:
We don't need that as we can sort server side by facet configuration using `"order" : { "_count" : "desc" }` or `"order" : { "_count" : "asc" }`. So we can remove this and use the same sort method for all facet directive

Reply:
@fxprunayre updated the pull request with your comments, thanks.